src/actions/lender.ts

```diff
@@ -17,6 +18,11 @@ export function LenderActions<ActionBase extends Constructor<TinlakeParams>>(Bas
       return waitAndReturnEvents(this.eth, txHash, this.contracts['SENIOR_OPERATOR'].abi, this.transactionTimeout);
     }
 
+    getSeniorTokenAllowance = async (owner: string) => {
+      const res: { 0: BN } = await executeAndRetry(this.contracts['SENIOR_TOKEN'].allowance, [owner, this.contractAddresses['SENIOR']]);
```

Comment:
I think we could just use one abi file for all erc20 token.

Like `this.contracts['ERC20']` for currency, senior token and junior token.

Reply:
Why are you `executeAndRetry` here? I thought `executeAndRetry` is only for transactions and not for calls (like allowance)?

Reply:
Good point, I have created a ticket for ABI cleanup.

Reply:
As discussed, `executeAndRetry` is agnostic to calls/transactions. It is used for calls as well since they also fall under Infura's rate limit.

Reply:
Yes, it makes sense.